appointment-booking-assistant/actions/schedule_appointment.py

```diff
@@ -0,0 +1,214 @@
+from typing import Any, Text, Dict, List
+from rasa_sdk import Action, Tracker
```

Comment:
Dont think any of the actions are required because we now use the out of the box react agent